audb/core/load.py

```diff
@@ -708,24 +746,44 @@ def database_cache_folder(
 
 
 def database_tmp_folder(
-        cache_root: str,
```

Comment:
Renamed to `db_root` since we usually refer to `cache_root` as the `audb' cache folder.

Reply:
Makes sense.